.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2
+
+jobs:
+  build:
+    working_directory: /workspace/go/src/istio.io/tools
+    docker:
+      # Image with KIND installed, based on Istio Kind node-image
+      # Should have all tools we need, recreated when we rev the tool versions.
+      - image: istionightly/kind:latest
+        environment:
+          GOPATH: /workspace/go
+          SKIP_CLEANUP: 1
+
+    # Can run multiple images, first is primary, all use localhost
+    steps:
+      - checkout
+      - setup_remote_docker: {}
+      # Only on paid version
+      #    docker_layer_caching: true
+      - run:
+          command: |
+            mkdir -p $GOPATH/src/istio.io
+            mkdir -p $GOPATH/src/github.com/istio-ecosystem
+            git clone https://github.com/istio/istio.git $GOPATH/src/istio.io/istio
+            git clone https://github.com/istio-ecosystem/istio-installer.git $GOPATH/src/github.com/istio-ecosystem/istio-installer
+      - run:
+          command: (cd $GOPATH/src/github.com/istio-ecosystem; make installer-test)
```

Comment:
Maybe I am missing the context here, but it looks like we are just testing the new installer here. Shouldn't this be tested in the new installer repo?

Reply:
It is testing the stability scripts (a subset for now) using the new installer.